src/server/index.js

```diff
@@ -54,9 +55,14 @@ async function createServer() {
 
   await server.register([sessionCache, nunjucksConfig])
 
-  // Register all of the controllers/routes defined in src/server/router.js
+  // Register all the controllers/routes defined in src/server/router.js
   await server.register([router])
 
+  server.events.on('stop', async () => {
+    server.logger.info(`Closing cache engine`)
+    await cacheEngine.stop()
```

Comment:
might not be needed.
hapi calls `cacheEngine.stop()` as part of `server.stop()`